6._Contributing.md

```diff
@@ -83,3 +85,10 @@ The next section contains more information on the workflow followed for Pull Req
 * Pull requests can be closed by the author without merging.
 
 * Pull requests may be closed by a Maintainer if the decision is made that it is not going to be merged.
+
+**4.6 Blog Post Specific**
+
+* A Community/Official Blog Post is considered to be a broad SLSA announcement that may be in regards to a new release, clarification, new tooling, showcase work, etc.  These blog posts are typically in coordination with the broader SLSA community and comes from full community consensus.  PRs must meet all necessary approvals from at least 2-3 steering committee members from differing organizations.  
```

Comment:
```suggestion
* A Community/Official Blog Post is considered to be a broad SLSA announcement that may be in regards to a new release, clarification, new tooling, showcase work, etc.  These blog posts are typically in coordination with the broader SLSA community and comes from full community consensus.  PRs must meet all necessary approvals from at least 2 steering committee members from differing organizations.  
```
It's better to be specific in how many reviewers we require. For [specification changes](https://github.com/slsa-framework/slsa/blob/main/CONTRIBUTING.md#review-and-approval) we require at least two steering committee members review, I propose we use the same language here.